secure/json-rpc/src/lib.rs

```diff
@@ -527,6 +524,19 @@ mod test {
     /// We only require implementing a subset of these API calls for testing purposes. To keep
     /// our code as minimal as possible, the unimplemented API calls simply return an error.
     impl DbReader for MockLibraDB {
+        fn update_to_latest_ledger(
```

Comment:
can't you put a default implementation in the trait that says unimplemented? especially for deprecated functions

Reply:
Sounds good. I forget this feature= =